emails/tests/models_tests.py

```diff
@@ -928,6 +929,40 @@ def test_profile_created_does_not_emit_metric_and_logs_from_measure_feature_usag
         mocked_incr.assert_not_called()
         mocked_events_info.assert_not_called()
 
+    @patch("emails.models.abuse_logger.info")
+    @patch("emails.models.datetime")
+    @override_settings(MAX_FORWARDED_PER_DAY=5)
+    def test_update_abuse_metric_flags_profile_when_abuse_threshold_met(
+        self, mocked_datetime, mocked_abuse_info
+    ):
+        expected_now = datetime.now(timezone.utc)
+        mocked_datetime.combine.return_value = datetime.combine(
+            datetime.now(timezone.utc).date(), datetime.min.time()
+        )
+        mocked_datetime.now.return_value = expected_now
+        mocked_datetime.side_effect = lambda *args, **kw: datetime(*args, **kw)
```

Comment:
Used the example on Python documentation here to mock `datetime.now` without getting an error! https://docs.python.org/3/library/unittest.mock-examples.html#partial-mocking